react/the_react_ecosystem/react_router.md

```diff
@@ -431,13 +435,13 @@ ReactDOM.createRoot(document.getElementById("root")).render(
 );
 ~~~
 
-This seems so much nicer, right?
+Much nicer! 
 
 ### Protected routes and navigation
 
-Often, you will face a need to decide when a certain route is rendered or not. One use case is authentication, where you may want to render certain routes based on if the user is logged in or not. If they are logged in, you may want to show some information about the user like [here at the dashboard](https://www.theodinproject.com/dashboard). While there are many ways to do so, one of the easiest ways can be to conditionally creating a config for the router.
+Often, you will need to decide whether a certain route should be rendered or not. One example is authentication, where you render certain routes based on if the user is logged in or not. If they are logged in, you show some information about the user like [here at the dashboard](https://www.theodinproject.com/dashboard). While there are many ways to do so, one of the easiest ways is to conditionally create a config for the router.
```

Comment:
I think this will read better and flow better to the next paragraph if we add an "Otherwise" after mentioning "If they are logged in". 

```suggestion
Often, you will need to decide whether a certain route should be rendered or not. One example is authentication, where you render certain routes based on if the user is logged in or not. If they are logged in, you show some information about the user like [here at the dashboard](https://www.theodinproject.com/dashboard). Otherwise, they are redirected to the sign-in page. While there are many ways to do so, one of the easiest ways is to conditionally create a config for the router.
```